google-datacatalog-kafka-connector/tests/google/datacatalog_connectors/kafka/prepare/assembled_entry_factory_test.py

```diff
@@ -13,10 +15,7 @@ class AssembledEntryFactoryTestCase(unittest.TestCase):
     __LOCATION_ID = 'location_id'
```

Comment:
Do we even need those constants if they are used only once, in the same file?

Reply:
We will probably need them later again, because there are more tests coming to this file, when I add tags creation

Reply:
okay, makes sense